bundles/org.openhab.binding.nanoleaf/src/main/java/org/openhab/binding/nanoleaf/internal/model/Rhythm.java

```diff
@@ -19,65 +19,65 @@
  */
 public class Rhythm {
 
-    private Boolean rhythmConnected;
+    private boolean rhythmConnected;
     private Boolean rhythmActive;
-    private Integer rhythmId;
+    private int rhythmId;
     private String hardwareVersion;
     private String firmwareVersion;
     private Boolean auxAvailable;
     private Integer rhythmMode;
     private RhythmPos rhythmPos;
 
-    public Boolean getRhythmConnected() {
+    public boolean getRhythmConnected() {
         return rhythmConnected;
     }
 
-    public void setRhythmConnected(Boolean rhythmConnected) {
+    public void setRhythmConnected(boolean rhythmConnected) {
         this.rhythmConnected = rhythmConnected;
     }
 
     public Boolean getRhythmActive() {
-        return rhythmActive;
+        return rhythmActive == null ? Boolean.FALSE : rhythmActive;
     }
 
     public void setRhythmActive(Boolean rhythmActive) {
         this.rhythmActive = rhythmActive;
     }
 
-    public Integer getRhythmId() {
+    public int getRhythmId() {
         return rhythmId;
     }
 
-    public void setRhythmId(Integer rhythmId) {
+    public void setRhythmId(int rhythmId) {
         this.rhythmId = rhythmId;
     }
 
     public String getHardwareVersion() {
-        return hardwareVersion;
+        return hardwareVersion == null ? "" : this.hardwareVersion;
     }
 
     public void setHardwareVersion(String hardwareVersion) {
         this.hardwareVersion = hardwareVersion;
     }
 
     public String getFirmwareVersion() {
-        return firmwareVersion;
+        return firmwareVersion == null ? "" : this.firmwareVersion;
     }
 
     public void setFirmwareVersion(String firmwareVersion) {
         this.firmwareVersion = firmwareVersion;
     }
 
     public Boolean getAuxAvailable() {
-        return auxAvailable;
+        return auxAvailable == null ? Boolean.FALSE : this.auxAvailable;
```

Comment:
Please check all fields if a primitive can be used (int and boolean) and change methods accordingly. Otherwise you get mix ups like in this method.

Reply:
Changed fields in model classes to primitives where possible